### PR TITLE
Add final airdrop amount and merkle tree, modded deploy scripts

### PR DESCRIPTION
**Pull request by HickupHH3**
- Final airdrop merkle tree added
- Deploy scripts modified to exclude token sale
- Added another deploy script for token sale and proposal submission to governor for
   - Changing tokenSale address in tokenLock
   - Transferring TOKEN_SALE_SUPPLY amount of arena tokens to token sale contract
   
 Edit: Looks like there's an issue with typechain generation. The current workaround is to omit one of the deploy scripts (eg. `./scripts/deployTokenSale`, do `yarn first-compile`, then re-include the omitted file.

### CODE REVIEW

**Comment by MrToph** [comment]
> Looks like there's an issue with typechain generation. The current workaround is to omit one of the deploy scripts (eg. ./scripts/deployTokenSale, do yarn first-compile, then re-include the omitted file.

had the same issue with another task, I had to create a new file with the typechain types and dynamically load it, see `scripts/proposals/index.ts`

File: scripts/deployTokenSale.ts [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  console.log(`waiting for block inclusion ...`);
+  await tx.wait(1);
+
+  console.log('exporting addresses...');
+  let addressesToExport = {
+    proposer: proposerAddress,
+    tokenSale: tokenSale.address,
+  };
+  let exportJson = JSON.stringify(addressesToExport, null, 2);
+  fs.writeFileSync(config.EXPORT_FILENAME, exportJson);
```
Comment (MrToph) [inline scripts/deployTokenSale.ts]: do we just want to merge the `tokenSale` with the other deployments in the existing `deployments/${config.EXPORT_FILENAME}` and write it? I don't think the proposer is important

**Review by MrToph — COMMENTED** [review]
Merkle tree doesn't have duplicate addresses.
Can we use the merkle json also in `test/fixtures/testAirdrop.json` to test if everyone on the list can claim?

**Comment by HickupHH3** [comment]
Makes sense, will do the replacement

**Review by MrToph — APPROVED** [review]

**Comment by tabshaikh** [comment]
LGTM deploy script refactored, added params for final mainnet deploy and merkle data

**Review by tabshaikh — APPROVED** [review]
LGTM deploy script refactored, added params for final mainnet deploy and merkle data

**Pull request merged**